public/app/plugins/panel/canvas/CanvasContextMenu.tsx

```diff
@@ -108,13 +109,23 @@ export const CanvasContextMenu = ({ scene, panel }: Props) => {
       const submenuItems: Array<
         React.ReactElement<MenuItemProps<unknown>, string | React.JSXElementConstructor<unknown>>
       > = [];
+
+      const onClickItem = (option: SelectableValue<string>) => {
+        let offsetY = 0;
+        if (scene.div) {
+          const sceneContainerDimensions = scene.div.getBoundingClientRect();
+          offsetY = anchorPoint.y - sceneContainerDimensions.top;
+        }
```

Comment:
Great job on this! Added a small change to offset y by scene container so added element aligns better with anchor point 


https://user-images.githubusercontent.com/22381771/210879864-fcf20fef-9a14-47cd-baca-c242735eba17.mov

Reply:
Thanks for the update 😁 I chose not to offset it as to me it looks a bit off, while using just the anchor point the item gets displayed right where the item was chosen from the list. Personal preference ofc, but it seems more natural to me 😁

Reply:
Ah I see your point there. My thinking was if we have it offset from anchor point there is potential for user to add an element that is off screen / vs. we can ensure added element is always on screen if we originate it at the anchor point for the context menu